src/NHSD.BuyingCatalogue.Solutions.API/Controllers/AdditionalServiceController.cs

```diff
@@ -12,20 +13,23 @@ public sealed class AdditionalServiceController : ControllerBase
     {
         [HttpGet]
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Performance", "CA1822:Mark members as static", Justification = "Cannot be viewed on swagger if static")]
-        public ActionResult<AdditionalServiceResult> Get([FromQuery] string[] solutionIds)
+        public ActionResult<List<AdditionalServiceResult>> Get([FromQuery] string[] solutionIds)
         {
             if (solutionIds is null)
                 return NotFound();
 
-            return new AdditionalServiceResult
+            return new List<AdditionalServiceResult>
```

Comment:
Iterate over the SolutionIds here, and add an AdditionalService for each one?

Reply:
👍